resources/ext/ezbake.conf

```diff
@@ -4,9 +4,9 @@ ezbake: {
     debian: { dependencies: ["pe-puppet (>= 4.0.0-1puppetlabs1)"] }
```

Comment:
Its probably going to change at some point anyway with AIO work. Today we pin everything within PE anyway, so this kind of thing matters very little, since the installer does the dependency handling outside of the packaging to some degree. I wouldn't stress that much about it, this will all be resolved when we solve the PE build problems for ezbake, and probably change for AIO anyway.